src/menu/kinstone_menu.c

```diff
@@ -65,7 +65,16 @@ Subtask KinstoneMenu_Type3;
 Subtask KinstoneMenu_Type4;
 Subtask KinstoneMenu_Type5;
 
-ASM_FUNC("asm/non_matching/menu/kinstone_menu/sub_080A3B48.inc", s32 sub_080A3B48(void))
+u32 sub_080A3B48(void) {
```

Comment:
Does this look better?
u32 sub_080A3B48(void) {
    u32 index;
    for (index = 0; index <= 0x12 && gSave.unk12B[index] != 0; index++) {
    }
    return index;
}

Not critical